services/movie.py

```diff
@@ -0,0 +1,42 @@
+from db.models import Movie, Genre, Actor
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> list[int]:
+    if genres_ids and actors_ids:
+        return Movie.objects.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
+    if genres_ids:
+        return Movie.objects.filter(genres__id__in=genres_ids)
+
+    if actors_ids:
+        return Movie.objects.filter(actors__id__in=actors_ids)
+
+    return Movie.objects.all()
```

Comment:
There is no need for multiple `return` statements, prefer using one at the end of your method and filter queryset during the function execution.